eq-author/src/App/settings/ThemesPage.js

```diff
@@ -123,34 +123,47 @@ const ThemesPage = ({ questionnaire }) => {
       </ValidationError>
     ));
 
+  const renderThemeErrors = (validationErrorInfo) =>
+    validationErrorInfo.errors.map(
+      ({ errorCode }, index) =>
+        errorCode === "ERR_FORM_TYPE_FORMAT" && (
+          <ValidationError key={index} right>
+            {THEME_ERROR_MESSAGES.ERR_FORM_TYPE_FORMAT}
+          </ValidationError>
+        )
+    );
```

Comment:
I would encapsulate this into the `FormType` component, as it's looking for an error specifically to that.

Maybe something like:

```
const findErrorsByCodePrefix = ({errors}, prefix) => 
    errors.map( ({errorCode, ...rest}) => 
        errorCode.contains(prefix) ?? {errorCode, ...rest})
```

Then, when you're using the `FormType` component on the themes page:

```
<FormType
              formType={formType}
              questionnaireId={id}
              shortName={shortName}
              errors={findErrorsByCodePrefix('ERR_FORM_TYPE')}
/>
```

Then, in the `FormType` component, you'd have something akin to your current `renderThemeErrors` function that maps over the errors passed into the `FormType` component and renders the specific styling.

Alternatively, if you want to keep your current set up (which is totally vali as it ensures all errors on the themes page are standardised), you could modify your `renderThemeErrors` like so:

```suggestion
  const renderThemeErrors = ({errors}, prefix) =>
    errors.map(
      ({ errorCode }, index) =>
        errorCode.contains("ERR_FORM_TYPE") && (
          <ValidationError key={index} right>
            {THEME_ERROR_MESSAGES.ERR_FORM_TYPE_FORMAT}
          </ValidationError>
        )
    );
```

As, then it can be used when we come to add validation onto other things in the page.